app/scripts/controllers/transactions/pending-tx-tracker.js

```diff
@@ -145,16 +145,16 @@ class PendingTransactionTracker extends EventEmitter {
     // If another tx with the same nonce is mined, set as failed.
```

Comment:
comment: "set as dropped" instead of "failed" ?

Reply:
oh woops i'll fix that